initialization:
  actionlib_server_names:
    mechanism_action_name: /realtime_loop/mechanism_controller
    move_action_name: /realtime_loop/move_controller
    client_action_name: mechanism_client
  controller_names:
    move_controller: move_joint_controller
    mechanism_controller: mechanism_joint_controller
  controller_timeouts:
    move_timeout: 30
    mechanism_timeout: 500
  gravity_compensation_service_name: /gravity_compensation/calibrate_bias
  server_timeout: 2
  exclude_controller_names: [base_controller, base_odometry, head_traj_controller, laser_tilt_controller, torso_controller, r_gripper_controller, l_gripper_controller, r_arm_controller, l_arm_controller, head_camera_trigger, l_forearm_cam_trigger, r_forearm_cam_trigger]

experiment:
  use_nullspace: true
  use_estimates: false
  nullspace_gain: 0.01
  init_t_error: 0.0
  init_k_error: 0.0
  feedback_rate: 1
  rod_arm: 1 # 0 - left; 1 - right
  surface_arm: 0
  base_link_name: base_link
  surface_arm_tool_frame: l_gripper_tool_frame
  num_of_experiments: 1
  bag_prefix: garbage
  vd_amp: 0.07
  vd_freq: 0.3
  wd_amp: 0.0
  wd_freq: 0.2
  goal_force: 2.0
  goal_torque: 0.01
  initial_pose:
    rod: [0.600, -0.13, 0.135, 0.573, 0.561, -0.419, 0.425] #[0.520, -0.10, 0.135, 0.573, 0.561, -0.419, 0.425]
    surface: [0.610, 0.271, -0.160, 0.018, -0.001, -0.708, 0.706] #[0.510, 0.327, -0.119, 0.204, 0.244, -0.655, 0.686]

  logging:
    toggle_logging_service: /data_logger/toggle